client/src/components/ViewModeToggle/ViewModeToggle.js

```diff
@@ -133,9 +129,10 @@ class ViewModeToggle extends Component {
     const { activeState, previewIconClass } = this.props;
 
     // Highlight if chosen view mode
-    const itemClass = classNames('btn', 'icon-view',
+    const itemClass = classNames(
+      'btn', 'icon-view', 'viewmode-toggle__button',
       previewIconClass,
-      { 'viewMode-selected': (activeState === 'preview') }
+      { 'viewmode-toggle--selected': (activeState === 'preview') }
```

Comment:
Shouldn't this be `viewmode-toggle__selected` for consistency?

Reply:
Well, `selected` is a modifier so it should use `--`